### DISCUSSION TITLE
How can I skip .class{ max-with: } only from generated css

### DISCUSSION THREAD

**Discussion by sytolk** [tailwindlabs/tailwindcss#11790] [Help] [answered]
Hello,
I have use sources of [milkdown demo](https://github.com/Milkdown/website) website. And there is generated css class `.prose{ max-width:65ch }` is it possible to easy skip this max-with properties only without to change milkdown sources. I have skip the whole .prose class from tailwind.config.js 
```
module.exports = {
  content: ['./src/**/*.{js,ts,jsx,tsx,css}'],
  darkMode: 'class',
  theme: {
    extend: {}
  },
  future: {
    hoverOnlyWhenSupported: true
  },
  plugins: [require('@tailwindcss/typography'), require('tailwind-nord')]
  blocklist: ['prose']
};
```
but this blacklist will remove all `.prose` styles like` .prose a`
How to skip or overwrite max-width only?

**Comment by wongjn** [comment, marked as answer]
As per the documentation, you could:

- [Override `max-width`](https://tailwindcss.com/docs/typography-plugin#overriding-max-width):
  > […] add `max-w-none` to your content to override the embedded max-width:
  >
  > ```html
  > <div class="grid grid-cols-4">
  >   <div class="col-span-1">
  >     
  >   </div>
  >   <div class="col-span-3">
  >     <article class="prose max-w-none">
  >       {{ markdown }}
  >     </article>
  >   </div>
  > </div>
  > ```
  [Tailwind Play example](https://play.tailwindcss.com/B2XPqSso9q)
- [Undo the declaration in configuration](https://tailwindcss.com/docs/typography-plugin#customizing-the-css):
  > If you want to customize the raw CSS generated by this plugin, add your overrides under the `typography` key in the `theme` section of your `tailwind.config.js` file:
  ```js
  typography: {
    DEFAULT: {
      css: {
        maxWidth: false,
      },
    },
  },
  ```
  [Tailwind Play example](https://play.tailwindcss.com/0AWYPYrX1n?file=config)

**Reply by sytolk** [reply]
Thanks @wongjn 
In first case I can set max-w-none of the parent div element but this cannot change style of the milkdown contenteditable div element: 
![image](https://github.com/tailwindlabs/tailwindcss/assets/1415463/bc27ce23-0157-4ec5-b578-412985df9482)

https://play.tailwindcss.com/2Rfy3ni3uk

adding typography section don't change generated css I have in tailwind.css `.prose{color:var(--tw-prose-body);max-width:65ch}`
```
module.exports = {
  content: ['./src/**/*.{js,ts,jsx,tsx,css}'],
  darkMode: 'class',
  theme: {
    extend: {}
  },
  future: {
    hoverOnlyWhenSupported: true
  },
  plugins: [require('@tailwindcss/typography'), require('tailwind-nord')],
  // blocklist: ['prose']
  typography: {
    DEFAULT: {
      css: {
        maxWidth: false
      }
    }
  }
};
```
my build css command:
`"build:css": "npx cross-env NODE_ENV=production tailwindcss -i src/globals.css -o src/tailwind.css -m",`

**Reply by wongjn** [reply]
In the first case, you'd want to add `max-w-none` class to the same element as the `prose` class or you could consider using an [arbitrary variant](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) to apply `max-width: none` to the element from some parent element.

For the configuration method, set `typography` in `theme.extend`, not in the root of the object:
```diff
 module.exports = {
   content: ['./src/**/*.{js,ts,jsx,tsx,css}'],
   darkMode: 'class',
   theme: {
-    extend: {}
+    extend: {
+      typography: {
+        DEFAULT: {
+          css: {
+            maxWidth: false
+          }
+        }
+      }
+    }
   },
   future: {
     hoverOnlyWhenSupported: true
   },
   plugins: [require('@tailwindcss/typography'), require('tailwind-nord')],
   // blocklist: ['prose']
-  typography: {
-    DEFAULT: {
-      css: {
-        maxWidth: false
-      }
-    }
-  }
};

**Reply by sytolk** [reply]
case 2 works! Thanks
I see now where is comming this style: https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/styles.js#L1239
